deployments/deploy-pxb-cloud.sh

```diff
@@ -333,6 +333,7 @@ if [ -n "${INTERNAL_DOCKER_REGISTRY}" ]; then
 fi
 
 kubectl create configmap cloud-config --from-file=/config/cloud-json
+kubectl create configmap cloud-source-config --from-file=/cluster/cloud-source-config
```

Comment:
Why are we creating this config map? Can you give a more meaningful name for this configmap?